source/code/go/src/plugins/oms.go

```diff
@@ -990,11 +992,28 @@ func InitializePlugin(pluginConfPath string, agentVersion string) {
 		} else {
 			Computer = strings.TrimSuffix(ToString(containerHostName), "\n")
 		}
+		// read proxyendpoint if proxy configured
+		ProxyEndpoint = ""
+		proxyConfigPath := pluginConfig["omsproxy_conf_path"]
+		if _, err := os.Stat(proxyConfigPath); err == nil {
+			Log("Reading proxy configuration for Linux from %s", proxyConfigPath)
+			proxyConfig, err := ioutil.ReadFile(proxyConfigPath)
```

Comment:
I see that in your oms docker PR, there is an env being set (PROXY_ENDPOINT). If so why can't you consume that rather then read from config file ?